lambda/lambda_handler.py

```diff
@@ -1,9 +1,44 @@
 """Handler."""
+from typing import Any, Dict, List
+
 from github_usage import process_event
 from logger import LOG
 
+LambdaDict = Dict[str, Any]
+
+
+class LambdaCognitoIdentity(object):
+    cognito_identity_id: str
+    cognito_identity_pool_id: str
+
+
+class LambdaClientContextMobileClient(object):
+    installation_id: str
+    app_title: str
+    app_version_name: str
+    app_version_code: str
+    app_package_name: str
+
+
+class LambdaClientContext(object):
+    client: LambdaClientContextMobileClient
+    custom: LambdaDict
+    env: LambdaDict
+
+
+class LambdaContext(object):
+    function_name: str
+    function_version: str
+    invoked_function_arn: str
+    memory_limit_in_mb: int
+    aws_request_id: str
+    log_group_name: str
+    log_stream_name: str
+    identity: LambdaCognitoIdentity
+    client_context: LambdaClientContext
```

Comment:
you almost certainly want these to be `dataclass`es https://docs.python.org/3/library/dataclasses.html

Reply:
Done.